src/python/grpcio/support.py

```diff
@@ -45,7 +45,9 @@
 """
 C_PYTHON_DEV_ERROR_MESSAGE = """
 Could not find <Python.h>. This could mean the following:
-  * You're on Ubuntu and haven't `apt-get install`ed `python-dev`.
+  * You're on Ubuntu and haven't run `apt-get install python-dev`.
+  * You're on RHEL/Fedora and haven't run `yum/dnf install python-devel`
```

Comment:
Gonna be a stickler (sorry!): while one'd expect folks to know that `yum/dnf` means `yum` or `dnf`, mind splitting the two into "`yum install python-devel` or `dnf install python-devel`"?